diff --git a/wasp/tests/aws-cryptosdk-c/stubs/aws_default_allocator_stub.c b/wasp/tests/aws-cryptosdk-c/stubs/aws_default_allocator_stub.c
index 5eda7dd..f88f7c4 100644
--- a/wasp/tests/aws-cryptosdk-c/stubs/aws_default_allocator_stub.c
+++ b/wasp/tests/aws-cryptosdk-c/stubs/aws_default_allocator_stub.c
@@ -15,6 +15,6 @@
 
 #include <proof_helpers/proof_allocators.h>
 
-struct aws_allocator aws_default_allocator() {
+struct aws_allocator *aws_default_allocator() {
     return can_fail_allocator();
 }
